docs/04-api-reference/front-commerce-core/graphql.mdx

```diff
@@ -0,0 +1,435 @@
+---
+title: "graphql"
+sidebar_position: 7
+description:
+  "This documentation cover methods related to GraphQL exposed by
+  `@front-commerce/core`."
+---
+
+<p>{frontMatter.description}</p>
+
+## `createGraphQLModule`
+
+Create a GraphQL module and inject it into Front-Commerce's GraphQL server
+
+```ts
+import { createGraphQLModule } from "@front-commerce/core/graphql";
+
+export default createGraphQLModule({
+  namespace,
+  modules,
+});
+```
+
+Arguments:
+
+- `namespace` (`string`): The namespace of the GraphQL module
+- `modules` (`modules[]`): The modules to inject in namespace
+
+_Example_
+
+```ts
+import { createGraphQLModule } from "@front-commerce/core/graphql";
+import AcmeModule from "./acmeModule";
+
+export default createGraphQLModule({
+  namespace: "Acme",
+  modules: [AcmeModule],
+});
+```
+
+## `readGraphQLModules`
+
+Read and parse GraphQLModules generated by codegen.
+
+_Example_
+
```

Comment:
**nit:** I think this is self-explanatory 😛 

**suggestion:** Remove this line.
```suggestion

```